docs/shop4cfcontext.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    
+    "isDefinedBy": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/isDefinedBy",
```

Comment:
I would correct these URLs to `.../shop4cf/data-models/master/...`. Is this ok?

Of course, they will start to work once this PR is merged.

Reply:
This is ok